### ISSUE TITLE
Borgs whisper in human understandable speech when they talk on the Binary radio channel

### ISSUE THREAD

**Issue by IProduceWidgets** [space-wizards/space-station-14#21367]
## Description

Silicons currently whisper in human understandable speech when they talk on the Binary radio channel. This means you can listen in to what a silicon is saying to other silicons when you should not be able to.

**Reproduction**

As Urist, stand next to a borg > borg talks on binary channel > Urist can heard the borg speak.

**Screenshots**

**Additional context**

This will impact MALF heavily when that is added.

**Comment by deltanedas** [comment]
yeah it shouldnt whisper anything since its not using speech synthesizer meme

**Comment by Errant-4** [comment]
A limitation to this is that unless the entire speech indicator system is upgraded, you will still see a borg was typing something, and then not say anything. Of course the player could just have changed their mind, or they might have radioed on a legitimate channel, but it will still be a warning sign
